src/PublicApiAnalyzers/UnitTests/DeclarePublicApiAnalyzerTests.cs

```diff
@@ -1224,6 +1224,51 @@ public struct Nested { }
                 );
         }
 
+        [Fact]
+        public async Task ShippedTextWithMissingImplicitStructConstructor()
+        {
+            var source = @"
+public struct C
+{
+}
+";
+
+            var shippedText = @"
+C";
+            var unshippedText = string.Empty;
+
+            // Test0.cs(2,15): warning RS0016: Symbol 'implicit constructor for C' is not part of the declared API.
```

Comment:
Why is there no code fix for this diagnostic? All `RS0016` must have a corresponding code fix with FixAll support, otherwise they become very onerous.